app/controllers/v1/sessions_controller.rb

```diff
@@ -211,13 +212,21 @@ def login_stats(status, saml_response, error = nil)
       tags = ["context:#{type}", VERSION_TAG]
       case status
       when :success
-        StatsD.increment(STATSD_LOGIN_NEW_USER_KEY, tags: [VERSION_TAG]) if type == 'signup'
+        if type == 'signup'
+          StatsD.increment(STATSD_LOGIN_NEW_USER_KEY, tags: [VERSION_TAG])
+          Rails.logger.info("LOGIN_NEW_USER_KEY, tags: [#{VERSION_TAG}]")
+        end
         # track users who have a shared sso cookie
         StatsD.increment(STATSD_LOGIN_SHARED_COOKIE, tags: tags)
+        Rails.logger.info("LOGIN_SHARED_COOKIE, tags: #{tags}")
```

Comment:
@benniemosher I don't think we need to add log messages for `LOGIN_NEW_USER_KEY, LOGIN_SHARED_COOKIE and LOGIN_LATENCY` right now.  For our purposes of debugging authentication `SSO_NEW_KEY, LOGIN_STATUS_SUCCESS and LOGIN_STATUS_FAILURE` will be fine

Reply:
This should be updated now.